quetz/condainfo.py

```diff
@@ -106,6 +107,9 @@ def _load_jsons(self, tar):
         self.info = json.load(tar.extractfile("info/index.json"))
         self.about = json.load(tar.extractfile("info/about.json"))
         self.paths = json.load(tar.extractfile("info/paths.json"))
+        with tar.extractfile("info/files") as fp:
```

Comment:
you may also need to extract files from .conda format packages

Reply:
Yes, this works. Just checked.
This is because the `_load_jsons` function is called after both the formats have been handled 👍🏼